src/plugins/header/header.tsx

```diff
@@ -33,16 +35,13 @@ export class UIPHeader extends UIPPlugin {
 
   /** Renders {@link UIPOptions} element */
   protected renderOptions(): void {
-    const optionsEl = document.createElement(UIPOptions.is) as UIPSnippets;
+    const optionsEl = document.createElement(UIPOptions.is) as UIPOptions;
     this.append(optionsEl);
   }
 
   /** Renders copy icon */
   protected renderCopy(): void {
-    const icon = document.createElement('button');
-    icon.title = 'copy markup';
-    icon.classList.add('copy-icon');
-    icon.append(UIPOptionIcons.copySVG);
+    const icon = <button title="copy markup" className="copy-icon">{UIPOptionIcons.copySVG.cloneNode(true) as HTMLElement}</button>;
```

Comment:
Do we really need to use `.cloneNode(true)`? `UIPOptionIcons.copySVG` returns new element on every call, so there shouldn't be any reasons to copy it.

Reply:
Actually... Yes) But not in the case of copy icon, but with `UIPOptions.UIPOptionsConfig` . The problem occurs when we have more than one playground on a page. When we copy link to an element from `UIPOptionIcons` to `UIPOptions.UIPOptionsConfig.svg` it behaves as a single instance. Append the second icon to the document removes previous one.

You can look it alive at our current github page https://exadel-inc.github.io/ui-playground/examples/example/multiple-playgrounds/